core/src/test/kotlin/api/RunsRouteIntegrationTest.kt

```diff
@@ -1816,11 +1727,16 @@ class RunsRouteIntegrationTest : AbstractIntegrationTest({
 
                 val statistics = response.body<OrtRunStatistics>()
 
-                statistics.issuesCount shouldBe 1
-                statistics.packagesCount shouldBe 2
-                statistics.ecosystems?.size shouldBe 2
-                statistics.vulnerabilitiesCount shouldBe null
-                statistics.ruleViolationsCount shouldBe null
+                with(statistics) {
+                    issuesCount.shouldBeNull()
```

Comment:
_nit_ isn't it `shouldBe null` ?
@sschuberth  right ?

+ other places

Reply:
Actually, it's `should beNull()` 😸

Reply:
Just out of curiosity is that a syntax choice, or is there some difference?

Reply:
Just syntax. In general, we prefer infix functions, i.e. those without the need for a `.` before the function call, to make the code read more fluent.